src/client.cpp

```diff
@@ -400,7 +400,7 @@ namespace konstructs {
                     request_chunks = 1;
                 }
                 for(auto it = received.begin(); it != received.end();) {
-                    if((*it - p_chunk).norm() > r) {
+                    if((*it - p_chunk).norm() > r + 1) {
```

Comment:
If we are at view distance 8 and drops down to 7, this change will make sure that we still have layer 8 loaded. If not we will drop payer 8 and ask for it a few ms later.